lib/lookup.json

```diff
@@ -72,7 +74,7 @@
   },
   "eslint": {
     "prefix": "v",
-    "flaky": ["ppc", "v7", "s390"],
+    "flaky": ["ppc", "v7", "s390", {"ubuntu": "16.04"}],
```

Comment:
technically we should, i'll double check but we are obtaining the version number so I figured that putting ubuntu and the version number in an object would work